server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/SpamAssassin.java

```diff
@@ -65,18 +72,18 @@
      * @see org.apache.mailet.base.GenericMailet#init()
      */
     public void init() throws MessagingException {
-        spamdHost = getInitParameter("spamdHost");
-        if (spamdHost == null || spamdHost.equals("")) {
-            spamdHost = "127.0.0.1";
+        spamdHost = getInitParameter(SPAMD_HOST);
+        if (StringUtils.isEmpty(spamdHost)) {
+            spamdHost = DEFAULT_HOST;
         }
 
-        String port = getInitParameter("spamdPort");
-        if (port == null || port.equals("")) {
-            spamdPort = 783;
+        String port = getInitParameter(SPAMD_PORT);
```

Comment:
```
spamPort = MailetUtil.getInitParameterAsStrictlyPositiveInteger(getInitParameter(SPAMD_PORT), DEFAULT_PORT);
```

Reply:
could we add a check to throw a `MessageException` when the port is higher that `65535`?